### ISSUE TITLE
Create get_archival_readings()

### ISSUE THREAD

**Issue by peterdesmet** [inbo/etn#210]
- [ ] SQL: define field names
- [ ] SQL: define field order
- [ ] SQL: make use of view
- [ ] Function: include SQL
- [ ] Function: update parameters (queries)
- [ ] Function: update documentation
- [ ] Tests: new order
- [ ] Tests: coverage
- [ ] Tests: pass

**Comment by PietrH** [comment]
This issue is unclear to me. Ok to close as unplanned?

**Comment by peterdesmet** [comment]
Yes, I think this can be closed as a duplicate of the later https://github.com/inbo/etn/issues/409
